app/src/main/java/org/pragyan/dalal18/fragment/StockExchangeFragment.kt

```diff
@@ -133,7 +136,7 @@ class StockExchangeFragment : Fragment() {
                             uiThread {
                                 if (response.statusCode == BuyStocksFromExchangeResponse.StatusCode.OK) {
                                     context?.toast("Stocks bought")
-                                    noOfStocksEditText.setText("")
+                                    noOfStocksEditText.setText("0")
```

Comment:
Is `if(noOfStocksEditText.text.toString()==""){` above required if you're adding this?

Reply:
Yes because the user can still type out the value in the edittext.